src/common/options.rs

```diff
@@ -1911,6 +1924,69 @@ impl OptionsBuilder {
         self
     }
 
+    /// Set the Keyterms feature.
+    ///
+    /// Calling this when already set will append to the existing keyterms, not overwrite them.
+    ///
+    /// See the [Deepgram Keyterms feature docs][docs] for more info.
+    ///
+    /// [docs]: https://developers.deepgram.com/docs/keyterm
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use deepgram::common::options::Options;
+    /// #
+    /// let options = Options::builder()
+    ///     .keyterms(["hello", "world"])
+    ///     .build();
+    /// ```
+    ///
+    /// ```
+    /// # use deepgram::common::options::Options;
+    /// #
+    /// let options1 = Options::builder()
+    ///     .keyterms(["hello"])
+    ///     .keyterms(["world"])
+    ///     .build();
+    ///
+    /// let options2 = Options::builder()
+    ///     .keyterms(["hello", "world"])
+    ///     .build();
+    ///
+    /// assert_eq!(options1, options2);
+    /// ```
+    pub fn keyterms<'a>(mut self, keyterms: impl IntoIterator<Item = &'a str>) -> Self {
+        self.0
+            .keyterms
+            .extend(keyterms.into_iter().map(String::from));
+        self
+    }
+
+    /// Add a single keyterm.
+    ///
+    /// This is a convenience method for adding a single keyterm.
+    /// For multiple keyterms, use [`OptionsBuilder::keyterms`] instead.
+    ///
+    /// See the [Deepgram Keyterms feature docs][docs] for more info.
+    ///
+    /// [docs]: https://developers.deepgram.com/docs/keyterm
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use deepgram::common::options::Options;
+    /// #
+    /// let options = Options::builder()
+    ///     .keyterm("hello")
+    ///     .keyterm("world")
+    ///     .build();
+    /// ```
+    pub fn keyterm(mut self, keyterm: impl Into<String>) -> Self {
```

Comment:
LGTM, but let's only expose a single method to set key terms. Let's keep the one above that takes an iterator.